apps/complete_registration/src/forms/lunch_tickets/lunch_ticket_create.page.tsx

```diff
@@ -0,0 +1,243 @@
+                    const errorData = await response.json();
+                    if (errorData && errorData.message) {
+                        setErrorString(errorData.message);
+                    } else {
+                        setErrorString("Unknown error");
+                    }
+                } catch (error) {
+                    console.error("Error parsing response:", error);
+                    setErrorString("Error parsing response");
+                }
+            } else {
+                // Reload: Not best practice, but I need to retrieve the token of the recently created ticket and place it on the queue
+                window.location.reload();
+            }
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        } catch (error:any) {
+            console.error("Error:", error);
+            const errorString = error.toString();
+            setErrorString(errorString);
+        }
+
+    }
+
+    return (
+        <FormWrapper className="flex flex-col gap-y-5 text-slate-700">
+            <h2 className="text-md text-center font-bold">
+                You can create
+                {(unassigned_lunch_tickets==0 ?
+                <Badge
+                    value={unassigned_lunch_tickets.toString()}
+                    severity="danger"
+                    className="mx-2"
+                />
+                :
+                <Badge
+                    value={unassigned_lunch_tickets.toString()}
+                    className="mx-2"
+                />
+                )}
+                lunch tickets
+            </h2>
+            <form
+                className="flex flex-col [&>div]:mb-4 [&>label]:pt-4 [&>label]:font-bold"
+                onSubmit={processForm}
+            >
+                <p className="text-red-500">{ErrorString}</p>
+                <div>
+                    <label htmlFor="ticket-date" className="pr-2 font-bold">
+                        Day:*
+                    </label>
+                    <Dropdown
+                        required={true}
+                        value={DateState}
+                        onChange={event => {
+                            setDateState(event.value)
+                        }}
+                        options={fair_days}
+                        className="md:w-14rem w-full"
+                    />
+                </div>
+                <div>
+                    <label
+                        htmlFor="ticket-date"
+                        className="block pr-2 font-bold"
+                    >
+                        Timeslot:*
+                    </label>
+                    <Dropdown
+                        required={true}
+                        value={TimeState}
+                        onChange={event => {
+                            setTimeState(event.value)
+                        }}
+                        options={lunch_times}
+                        className="md:w-14rem w-full"
```

Comment:
You were right, its incorrect, and therefore the css rules where not being applied. I deleted it instead since the styling on the ticket creation page is good enough